syft/frameworks/torch/he/fv/context.py

```diff
@@ -0,0 +1,21 @@
+class Context:
```

Comment:
What's the purpose of the context class if it's only holding the parameters?

Reply:
This class will store more attributes in future as the work progress.